pkg/controller/raven/gateway/gateway_controller.go

```diff
@@ -51,13 +51,9 @@ var (
 	controllerKind = ravenv1alpha1.SchemeGroupVersion.WithKind("Gateway")
 )
 
-const (
-	controllerName = "Gateway-controller"
-)
-
 func Format(format string, args ...interface{}) string {
 	s := fmt.Sprintf(format, args...)
-	return fmt.Sprintf("%s: %s", controllerName, s)
+	return fmt.Sprintf("%s: %s", common.ControllerName, s)
```

Comment:
Perhaps we could output it like this? `fmt.Sprintf("%s: %s", fmt.Sprintf("%s-gateway", common.ControllerName), s)`
This should help us more effectively to know exactly which sub-controller is outputting the logs.

Reply:
fixed